docs/glossary.md

```diff
@@ -0,0 +1,97 @@
+# Glossary
+
+Definition of commonly used terms in the context of this Romcal project. The terms are listed in alphabetical order.
```

Comment:
For the acronyms/abbreviation, I think it's important to keep them in the title to keep them prominent (especially GIRM, GNLYC, GILH). Since these acronyms are what we'll use elsewhere to refer to this document, readers of the documentation need to quickly associate the definition with the acronym. What do you think?

For adding the word `Liturgical` in front of several other words, why not? If it might cause confusion if `Liturgical` isn't present. Ideally, the term used should match the corresponding `type`/`class`/`enum` in the codebase. So we need something explicit, but not too long.

For the rest, I'm aligned with you.

Reply:
By the way, I applied your suggestion to change `Rank` to `Liturgical Rank`. However, upon reflection, I find it a bit verbose.

Reply:
> For the acronyms/abbreviation, I think it's important to keep them in the title to keep them prominent (especially GIRM, GNLYC, GILH). Since these acronyms are what we'll use elsewhere to refer to this document, readers of the documentation need to quickly associate the definition with the acronym. What do you think?

While I agree that (at least) those three initialism are quite important to have them in the glossary, however, I disagree to put them to the headings of the respective glossary items. I propose to add a separate glossary items for these (see also [this comment of mine](https://github.com/romcal/romcal/pull/586#discussion_r1659886469)).

> For adding the word `Liturgical` in front of several other words, why not? If it might cause confusion if `Liturgical` isn't present.

Unlike for the problem discussed above, I don’t consider removing (not adding) the `liturgical` word to any of the mentioned words a big issue. Still, I think it would be better to add it to some/most of the mentioned words because the words have multiple meanings or usages in different context, however, I get it that romcal defines the liturgical context.

Therefore, add `Liturgical` where you agree with me, remove it where you disagree with me. :wink:

> Ideally, the term used should match the corresponding `type`/`class`/`enum` in the codebase.

I think it should be the opposite way: the names of code types/classes/enums/variables should match the name used by the Church in (American) English nowadays. :wink: Where there are discrepancies, we should update the code, not the terms. :smiley:

> So we need something explicit, but not too long.

IMHO, the glossary item names should be short, but sufficiently specific.

> For the rest, I'm aligned with you.

Thanks! :pray:

I’ll re-review the glossary and add some more suggestions. :wink: